ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/minimal-permission/cucushift-installer-rehearse-aws-ipi-minimal-permission-workflow.yaml

```diff
@@ -4,6 +4,6 @@ workflow:
     pre:
     - chain: cucushift-installer-rehearse-aws-ipi-minimal-permission-provision
     post:
-    - chain: cucushift-installer-rehearse-aws-ipi-minimal-permission-deprovision
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
```

Comment:
We still need to run `aws-deprovision-users-and-policies` in deprovision phase, right?